egs2/aaf_openslr57/asr1/run.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+
+train_set="train"
+train_dev="valid"
+test_set="valid test"
+
+asr_config=conf/tuning/train_asr_conformer5.yaml
+inference_config=conf/decode_asr.yaml
+
+nbpe=150
+
+./asr.sh \
+    --ngpu 1 \
+    --nj 1 \
```

Comment:
could you remove this line, the default nj seems to be too small for efficiency issue